### DISCUSSION TITLE
Stricter filters increasing differentially expressed proteins?

### DISCUSSION THREAD

**Discussion by Mrfa0811** [Nesvilab/FragPipe-Analyst-doc#31] [Q&A] [answered]
Hi,

I'm using Fragpipe Analyst to run DIA analysis on a protein dataset. I've noticed that when I increase the value of the "Min percentage of non-missing values globally" or "Min percentage of non-missing values in at least one condition" under advanced settings (say, from 0 to 50), it is increasing the number of significant differentially expressed proteins. As I understand it these options are both filters, and thus should either reduce the number of differentially expressed proteins or have no effect. Is my understanding of the setting incorrect or is something else going on here?

Thanks!

**Comment by hsiaoyi0504** [comment, marked as answer]
@Mrfa0811 I don't think there is no statistical guarantee of how many differential expressed genes or proteins if you increase the filter. The factor you didn't mention is about the multiple testing correction. The strict filter could lead to less features subjected to the differential expression analysis, but then it might pass multiple testing correction easier.

**Reply by Mrfa0811** [reply]
Thank you, that makes sense! You are correct that I forgot how the stricter filter would alter the FDR corrections.
